src/samplers/cpu/linux/usage/bpf.rs

```diff
@@ -224,12 +224,17 @@ impl CpuUsage {
 
         let now = Instant::now();
 
+        let mut total_busy = Counter::new(&CPU_USAGE_BUSY, Some(&CPU_USAGE_BUSY_HISTOGRAM));
+        let mut total_idle = Counter::new(&CPU_USAGE_IDLE, Some(&CPU_USAGE_IDLE_HISTOGRAM));
```

Comment:
The lazy initialization is so that we are not exporting 0 value counts for metrics that are not supported. I don't think that we would want to initialize to 0 for all counters anway. For instance, if we're using the non-BPF CPU usage sampler, it basically reads values that the kernel is already tracking and will start at some positive value N. If we eagerly initialized the Rezolus counter to 0, we would have a jump and a corresponding spike in the histogram data. This would happen with every Rezolus restart.

Reply:
I'm not sure I understand the second point -- if we don't eagerly initialize it to 0, won't be have a random uninitialized value in that counter which might (or might not) lead to a spike anyway?

Reply:
Chatted offline; resolving.